include/openthread/platform/radio.h

```diff
@@ -361,6 +361,12 @@ typedef struct otRadioFrame
             bool mCsmaCaEnabled : 1;       ///< Set to true to enable CSMA-CA for this packet, false otherwise.
             bool mCslPresent : 1;          ///< Set to true if CSL header IE is present.
             bool mIsSecurityProcessed : 1; ///< True if SubMac should skip the AES processing of this frame.
+
+            /**
+             * The time of the local radio clock in microseconds when the end of
+             * the SFD was present at the local antenna.
+             */
+            uint64_t mTimestamp;
```

Comment:
Radio platform may handle frame retx. 
May help to clarify that `mTimestamp` should be for the last tx attempt?

Reply:
added more description. I think we should require the platform the update the field for each transmit attempt.